build.gradle

```diff
@@ -11,6 +11,12 @@ targetCompatibility = JavaVersion.VERSION_11
 
 def ROBOT_MAIN_CLASS = "frc.robot.Main"
 
+tasks.register('copyAutos', Copy) {
+    from "Autos"
+    into "src/main/deploy/Autos"
+}
```

Comment:
Does the Copy task gracefully handle when the file already exists? The documentation doesn't really seem to say https://docs.gradle.org/current/userguide/working_with_files.html